src/utils/Individual-Tweaks.psm1

```diff
@@ -426,6 +487,21 @@ function Enable-WindowsSearch() {
     Start-Service "WSearch"
 }
 
+function Disable-WindowsSpotlight() {
+    Write-Status -Types "-", "Privacy" -Status "Disabling Windows Spotlight Features..."
+    Set-ItemPropertyVerified -Path "$PathToCUPoliciesCloudContent" -Name "ConfigureWindowsSpotlight" -Type DWord -Value 2
+    Set-ItemPropertyVerified -Path "$PathToCUPoliciesCloudContent" -Name "IncludeEnterpriseSpotlight" -Type DWord -Value 0
+    Set-ItemPropertyVerified -Path "$PathToCUPoliciesCloudContent" -Name "DisableWindowsSpotlightFeatures" -Type DWord -Value 1
+    Set-ItemPropertyVerified -Path "$PathToCUPoliciesCloudContent" -Name "DisableWindowsSpotlightOnActionCenter" -Type DWord -Value 1
+    Set-ItemPropertyVerified -Path "$PathToCUPoliciesCloudContent" -Name "DisableWindowsSpotlightOnSettings" -Type DWord -Value 1
+    Set-ItemPropertyVerified -Path "$PathToCUPoliciesCloudContent" -Name "DisableWindowsSpotlightWindowsWelcomeExperience" -Type DWord -Value 1
+}
+
+function Enable-WindowsSpotlight() {
+    Write-Status -Types "*", "Privacy" -Status "Enabling Windows Spotlight Features..."
+    Set-ItemPropertyVerified -Path "$PathToCUPoliciesCloudContent" -Name "DisableWindowsSpotlightFeatures" -Type DWord -Value 0
```

Comment:
This registry key needs to be completely deleted or the Windows Settings app keeps Spotlight disabled and says "managed by your organization"
The other registry keys can exist without causing issues, but their values need to be inverted from those used in the `Disable-WindowsSpotlight` function.

Reply:
Yes, all the values need to be removed, I've created a new user and the path doesn't even exist. Updating as a hotfix.